modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -4590,7 +4809,37 @@ else if (Objects.equals(
 
 				Assert.assertEquals(
 					_getTimestamp(String.valueOf(excpectedValue.getValue())),
-					actualValues.get(excpectedValue.getKey()));
+					_getTimestamp(
+						String.valueOf(
+							actualValues.get(excpectedValue.getKey()))));
+			}
+		}
+		else if (Objects.equals(
+					expectedEntry.getKey(),
+					"localizedDateTimeObjectFieldName_i18n")) {
+
+			Map<String, Object> actualValues =
+				(Map<String, Object>)actualObjectEntryProperties.get(
+					expectedEntry.getKey());
+			Map<String, Object> expectedValues =
+				(Map<String, Object>)expectedEntry.getValue();
+
+			for (Map.Entry<String, Object> excpectedValue :
+					expectedValues.entrySet()) {
```

Comment:
`excpectedValue` > should be `expectedValues`

This same error is in the condition before this one ("localizedDateObjectFieldName_i18n").

Reply:
BTW, these conditions ons are sorted. How "localizedDateObjectFieldName_i18n" and "localizedDateTimeObjectFieldName_i18n" are at the end?
Can you sort it, please?

Reply:
Done!